packages/kolibri/utils/i18n.js

```diff
@@ -157,6 +157,7 @@ class Translator {
     this._defaultMessages = defaultMessages;
     for (const key in defaultMessages) {
       this[`${key}$`] = this.$tr.bind(this, key);
+      this[`${key}$`].KOLIBRI_I18N_WRAPPED_STRING == true;
```

Comment:
I found it :) this should be 
`this[`${key}$`].KOLIBRI_I18N_WRAPPED_STRING **=** true`

because we're assigning, not checking the value.

Reply:
TThank you for helping me spot it . I fixed it